addons/website/models/website.py

```diff
@@ -1474,7 +1481,17 @@ def _search_get_details(self, search_type, order, options):
 
         :return: list of search details obtained from the `website.searchable.mixin`'s `_search_get_detail()`
         """
-        result = []
+        if SEARCH_TYPE_MODELS:
+            # TODO: Remove in master.
+            if search_type == 'all':
+                model_names = reduce(operator.ior, SEARCH_TYPE_MODELS.values())
+            else:
+                model_names = SEARCH_TYPE_MODELS[search_type]
+            result = list(self.env[model_name]._search_get_detail(self, order, options)
+                    for model_name in model_names)
```

Comment:
Linter error, add some spaces

Reply:
Not sure what rule my linter is missing. I added an empty line after the `else` block, and split the list-comprehension on several lines, following blacks's suggestion.